### PR TITLE
Test Swatches: Reduce length of param list by checking for element type

### PR DESCRIPTION
**Pull request by kaalleen**
...

The list was a bit too long. I'm afraid users would curse until they find the param they were looking for otherwise.

### CODE REVIEW

File: lib/extensions/test_swatches.py [hunk 1]
```diff
@@ -5,51 +5,103 @@
...
 from ..i18n import _
-from ..svg.tags import EMBROIDERABLE_TAGS
+from ..utils.param import ParamOption
 from .base import InkstitchExtension
 
 
 class TestSwatches(InkstitchExtension):
     '''
-    This generates swatches from selection by altering one param each time.
+    This saves embroider settings into the metadata of the file
```
Comment (lexelby) [inline lib/extensions/test_swatches.py:17]: Is this correct?

File: lib/gui/test_swatches.py [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+        info_sizer.Add((20, 20), 0, 0, 0)
+
+        more_info_label = wx.StaticText(self.info, wx.ID_ANY, _("Get more information on our website"))
+        info_sizer.Add(more_info_label, 0, wx.ALL, 8)
+
+        self.help_link = wx.adv.HyperlinkCtrl(
+            self.info,
+            wx.ID_ANY,
+            "https://inkstitch.org/docs/edit/#generate-test-swatches-from-selection",
```
Comment (lexelby) [inline lib/gui/test_swatches.py:104]: Is it okay that this links to the English version for everyone?

**Review by lexelby — APPROVED** [review]
Aside from my questions, this looks great.  It makes sense to have this be a full app.

I like the idea of cursing-driven UI design ;)

File: lib/extensions/test_swatches.py (hunk 1 above)
Comment (kaalleen) [reply lib/extensions/test_swatches.py]: Hups, how did that happen. Thanks for pointing this out.

File: lib/gui/test_swatches.py (hunk 2 above)
Comment (kaalleen) [reply lib/gui/test_swatches.py]: Ah no :)

**Pull request merged**